docs/content/en/writing.md

```diff
@@ -92,7 +92,9 @@ It will be transformed to it's JSON AST structure, and by using the `nuxt-conten
 
 ### Links
 
-Links are transformed to add valid `target` and `rel` attributes. You can change this behaviour, see [configuration](/configuration#markdownexternallinks). Relative links are also automatically transformed to `nuxt-link` to provide navigations between page components and enhance performances with smart prefetching.
+Links are transformed to add valid `target` and `rel` attributes using [remark-external-links](https://github.com/remarkjs/remark-external-links). You can check [here](/configuration#markdown) to learn how to configure this plugin.
+
+Relative links are also automatically transformed to `nuxt-link` to provide navigations between page components and enhance performances with smart prefetching.
```

Comment:
Suggesting to use `[<nuxt-link>](https://nuxtjs.org/api/components-nuxt-link/)`

Reply:
I've used [nuxt-link] instead of [<nuxt-link>]